zabbix_smartmontools/__init__.py

```diff
@@ -178,7 +178,14 @@ def getSerial(device):
                 serial = match.groups()[1]
                 break
     else:
-        raise NotImplementedError
+        # On other platforms, we have to invoke smartctl, which is slow :(
+        cp = subprocess.run([config['ctlPath'], "-i", "/dev/%s" % device],
```

Comment:
```suggestion
        cp = None
        try:
            cp = subprocess.check_output([config['ctlPath'], "-i", "/dev/%s" % device],
                stderr=subprocess.PIPE,
                universal_newlines=True,
                timeout=3)
        except subprocess.CalledProcessError as e:
            cp = e.output
```
Numerous things could go wrong.